doc/classes/Skin.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Skin" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A list of inverse bind poses and corresponding bones.
 	</brief_description>
 	<description>
+		[Skin] contains a list of bind pose matrices and a list of either bone names if bones are named or bone indices if bones are not named. The bind pose matrix, also called offset matrix or inverse bind matrix, is a [Transform3D] offset that is multiplied with a bone's transform to get from vertex space to bone space. Used by [Skeleton3D] and [GPUParticles3D].
```

Comment:
Someone asked me why in a direct message for some context on the skins.

One problem skins is solving is we want to have a skeleton with associated skinned base meshes and another skeleton containing non-base meshes. We want to transfer the "non-base" meshes onto the "base" mesh.

The meshes only store the skin index which maps the vertex bone influence index to the skeleton joint name.

Provide a skeleton with the mesh and the mapping is kept.